packages/design-system/src/components/List/List.stories.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+export default {
+  title: 'Components/List',
```

Comment:
One idea for separating out List is that you could change the title to be 'Utilities/List' and that would put it in a separate grouping than other React components

I'm not sure if "utilities" is the term we want to use, but we could use the title to organize things that aren't quite components

Reply:
Agree. I talked to @line47 and he mentioned that this should go under a "Typography" section - so I'm going to change the title to reflect that.